docs/modules/layers.rst

```diff
@@ -127,6 +127,25 @@ Layer classes: embedding layers
 -------------------------------
 
 .. autoclass:: EmbeddingLayer
```

Comment:
You've inadvertently removed `:members:` here. (But don't worry too much. If #335 gets merged before this one, it will have to look a little different anyway.)